### PR TITLE
Add default settings to pass to chromecast and increment version

### PR DESCRIPTION
**Pull request by tsingh777**
https://github.com/KhalisFoundation/sttm-chromecast/pull/25 will catch issues in current prod. Need to confirm with @inderpreetsingh the source of truth for the language settings.

### CODE REVIEW

**Comment by inderpreetsingh** [comment]
looks good to me, will test it once on chromecast and approve. 🙏

File: www/js/defaults.json [hunk 1]
```diff
@@ -33,6 +33,10 @@
       "vishraam": {
         "vishraam-options": "colored-words",
         "vishraam-source": "sttm"
+      },
+      "language-settings": {
```
Comment (tsingh777) [inline www/js/defaults.json]: @inderpreetsingh this is duplicated from the `slide-layout` preference. which is needed?

Comment (inderpreetsingh) [reply www/js/defaults.json]: Yes its supposed to only be in slide-layout and that is needed. We access the settings in codebase as `store.getUserPref('slide-layout.language-settings.translation-language');`

Comment (inderpreetsingh) [reply www/js/defaults.json]: I am sorry if I am missing something obvious but I am unable to understand why this change was made. Can you please write a line or two explaining the issue?

Comment (tsingh777) [reply www/js/defaults.json]: Chromecast was looking at the `pref.toolbar` language setting. https://github.com/KhalisFoundation/sttm-chromecast/blob/master/src/js/index.js#L62 

I updated to chromecast to look for `prefs['slide-layout']` if `pref.toolbar` isn't there. I don't know if there was a version that set `pref.toolbar` for language settings. If there isn't one then we can clean up chromecast to not look for `pref.toolbar` at all and decline this PR.

Comment (inderpreetsingh) [reply www/js/defaults.json]: I will look into that. Thank you for addressing the chromecast fix 🙏

**Pull request merged**